Library/Formula/norm.rb

```diff
@@ -0,0 +1,30 @@
+class Norm < Formula
+  desc "NACK-Oriented Reliable Multicast"
+  homepage "https://www.nrl.navy.mil/itd/ncs/products/norm"
+  url "https://downloads.pf.itd.nrl.navy.mil/norm/archive/src-norm-1.5b3.tgz"
```

Comment:
All the releases are marked with 'b', so no AFAIK. zeromq requires a version >=1.5b3

I'm in touch with the maintainer, but not sure I can convince him to make a non 'b' release, is that a deal breaker for inclusion in homebrew?